signer/src/transaction_coordinator.rs

```diff
@@ -214,14 +222,22 @@ where
 
         let transaction_package = pending_requests.construct_transactions()?;
 
-        for transaction in transaction_package {
+        for mut transaction in transaction_package {
             self.sign_and_broadcast(
                 bitcoin_chain_tip,
                 aggregate_key,
                 signer_public_keys,
-                transaction,
+                &mut transaction,
             )
             .await?;
+
+            // TODO: need to add a retry (somewhere else?) if this fails? Otherwise
+            // since we already submitted the bitcoin tx, we would never mark the request
+            // as accepted.
+            self.context
+                .get_emily_client()
```

Comment:
> I was thinking about the case of Emily being offline or unreachable: since we already submitted the tx we would never mark it as accepted. That's why I was thinking of adding the retry in some other place, looking for inconsistencies.

I'll remove the comment though, since I guess it's the general _make it robust_ thing.

Reply:
Realistically what we should probably do here (long-term)  is store all emily "updates" in the db as a persistent queue and push them from there in a separate task. Since it's only the signer coord that sends these, and we don't know how long emily could be unreachable (could be a local network issue), that would help us avoid eating memory storing these in ram and offer better reliability in-case the signer restarts.